addons/l10n_ar_withholding/wizards/l10n_ar_payment_register_withholding.py

```diff
@@ -38,12 +64,38 @@ def _tax_compute_all_helper(self):
         tax_amount = taxes_res['taxes'][0]['amount']
         tax_account_id = taxes_res['taxes'][0]['account_id']
         tax_repartition_line_id = taxes_res['taxes'][0]['tax_repartition_line_id']
+
+        if self.tax_id.l10n_ar_tax_type in ['earnings_withholding', 'earnings_withholding_scale']:
+            # si es escala de ganancias calculamos segun escala
+            if self.tax_id.l10n_ar_tax_type == 'earnings_withholding_scale':
+                scale_name = 'Escala Normal' if self.tax_id.l10n_ar_code != '119' else 'Escala 119'
+                escala = self.env['l10n_ar.earnings.scale.detail'].search([
+                    ('scale_id.name', '=', scale_name),
```

Comment:
?????? Do not we have the many2one on the tax?

Reply:
@jco-odoo yes we do